### ISSUE TITLE
Support for Homebrew on Linux (amd64 and aarch64)

### ISSUE THREAD

**Issue by porelli** [autozimu/unison-fsmonitor#21]
Similar to #12 

When adding the formula and installing the package, the system receives the MacOS binary that of course cannot be executed.

```
$ file /home/linuxbrew/.linuxbrew/Cellar/unison-fsmonitor/0.3.0/bin/unison-fsmonitor
/home/linuxbrew/.linuxbrew/Cellar/unison-fsmonitor/0.3.0/bin/unison-fsmonitor: Mach-O 64-bit x86_64 executable, flags:<NOUNDEFS|DYLDLINK|TWOLEVEL|PIE|HAS_TLV_DESCRIPTORS>
```

Do you have any plan to support different platforms/architecture?

Thanks!

**Comment by porelli** [comment]
I added support for Linux and macOS for both amd64 and aarch64 here:
https://github.com/porelli/homebrew-formulas
https://github.com/porelli/unison-fsmonitor

There is still some work to do, for example reduce the size of the Linux binaries, but the macOS M1 release should work.

To test:
```
brew install porelli/homebrew-formulas/unison-fsmonitor
```

I do not have a Mac M1 to test the release, any help would be appreciated

**Comment by porelli** [comment]
I tested a little bit the Linux releases and it seems like it's not working. However, I tried also the binary generated directly with ```cargo install unison-fsmonitor``` and it has similar size and still doesn't work. The binary causes a huge spike in CPU usage at the beginning and then it remains at a constant 10-15% holding unison to perform anything.
Interestingly, the tests are passing without raising errors.

@autozimu - I know you mentioned that it should work but I guess it has not been properly tested?

Meanwhile, I opened a pull request to include unison-fsmonitor in the unison bottle for Linux: https://github.com/Homebrew/homebrew-core/pull/115779
